test/WebSites/RoutingWebSite/Program.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace RoutingWebSite
+{
+    public class Program
+    {
+        public const string EndpointRoutingScenario = "endpointrouting";
+        public const string RouterScenario = "router";
+
+        public static void Main(string[] args)
+        {
+            var webHost = GetWebHostBuilder(args).Build();
+            webHost.Run();
+        }
+
+        // For unit testing
```

Comment:
Oh realyl?

Reply:
Yeah. Functional tests pass an arg for the desired startup. Could be improved to do what MVC does.